src/worker/startup.ts

```diff
@@ -123,22 +124,13 @@ export async function start() {
 	// exposing queue metrics at a regular interval
 	timer = setInterval(sendQueueMetrics, 60000);
 
+	backfillSupplier.setRedisQueue(queues.installation);
+
 	// Start processing queues
 	queues.discovery.process(5, commonMiddleware(discovery(app, queues), DISCOVERY_LOGGER_NAME));
 	queues.installation.process(
 		Number(CONCURRENT_WORKERS),
-		commonMiddleware(processInstallation(app, () => {
```

Comment:
Copy-pasted as it was to https://github.com/atlassian/github-for-jira/pull/945/files#diff-a8e00f3426cb4fef101380799e36d85e06f59889ec8f9a3d147e6a38e959b977R35